docs/releases/1.10.txt

```diff
@@ -64,6 +64,8 @@ Minor features
 * :ref:`Distance lookups <distance-lookups>` now accept expressions as the
   distance value parameter.
 
+* Support for SpatiaLite < 3.0 has been dropped.
```

Comment:
A minor thing that nags me in the past release notes.... for consistency with the "Features removed" section, let's use "is" instead of "has been".